tests/end_to_end/http_server/large_files/src/index.ts

```diff
@@ -0,0 +1,112 @@
+import express, { Request } from 'express';
+import {
+    access,
+    copyFile,
+    mkdir,
+    readFile,
+    rename,
+    stat,
+    unlink,
+    writeFile
+} from 'fs/promises';
+
+import { ls, lsHtml, tree } from './list_files';
+
+const app = express();
+app.use(express.json());
+
+app.get(
+    '/exists',
+    async (req: Request<any, any, any, { path: string }>, res) => {
+        try {
+            await access(req.query.path);
+            res.send(true);
+        } catch {
+            res.send(false);
+        }
+    }
+);
+
+app.get('/size', async (req: Request<any, any, any, { path: string }>, res) => {
+    const size = await getFileSize(req.query.path);
+    res.json(size);
+});
+
+app.get(
+    '/ls',
+    async (
+        req: Request<any, any, any, { path: string; recursive: boolean }>,
+        res
+    ) => {
+        res.send(`<pre>${await ls(req.query.path, req.query.recursive)}</pre>`);
+    }
+);
+
+app.get(
+    '/lsHtml',
+    async (
+        req: Request<any, any, any, { path: string; recursive: boolean }>,
+        res
+    ) => {
+        res.send(`${await lsHtml(req.query.path, req.query.recursive)}`);
+    }
+);
+
+app.get('/tree', async (req: Request<any, any, any, { path: string }>, res) => {
+    res.send(`<pre>${await tree(req.query.path)}</pre>`);
+});
+
+app.get(
+    '/read-file',
+    async (req: Request<any, any, any, { path: string }>, res) => {
+        const content = await readFile(req.query.path, 'utf8');
+        res.send(content);
+    }
+);
+
+app.post('/write-file', async (req, res) => {
```

Comment:
We didn't end up using or testing these endpoints thinking that it might be best to lest ic-wasi-polyfill continue to mature on it's own and we just make sure that our previous issues were resolved. If that's the case do we want to get rid of all oft these other unused endpoints?

Reply:
As a matter of fact we have these in the fs example so I'm just going to get rid of them

Reply:
done